src/components/RadioPanel/RadioPanel.tsx

```diff
@@ -0,0 +1,65 @@
+import React, { ReactNode, Ref, forwardRef } from 'react'
+import { Panel, PanelBody, PanelContent } from '../Panel'
+import styled from '@emotion/styled'
+import { Radio, RadioProps } from '../RadioGroup/Radio'
+import { fontWeight, space } from '../../theme'
+import { Label } from '../Input'
+
+export interface RadioOption
+  extends Omit<RadioProps, 'id' | 'value' | 'label'> {
+  id: string
+  value: string
+  label: ReactNode
+  description?: () => ReactNode
```

Comment:
Why did you make this a function? Instead of just `ReactNode` 🤔

Reply:
No need indeed, fixed!